demo/admin/package.json

```diff
@@ -1,27 +1,4 @@
 {
-    "name": "comet-demo-admin",
```

Comment:
What I liked about the previous rule was that it sorted package.json in "logical" order (name, version, scripts etc.) instead of alphabetical order. Could we revert that?

Reply:
i think its not possible to logically sort this jsons.

Reply:
Then we should remove sorting jsons altogether. IMO alphabetical sorting is a downgrade for package.json and tsconfig.json

Reply:
adding the package `eslint-plugin-package-json`  and configuring it accordingly is now linting the package.json. And it also sorts the collection ("scripts", "dependencies",...) alphabetically and keeps the collections in a logical order.